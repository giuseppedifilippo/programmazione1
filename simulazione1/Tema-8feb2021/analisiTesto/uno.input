È Bëatrice quella che sì scorge
  di bene in meglio, sì subitamente
    che l’atto suo   per tempo non si sporge.

		Quant’ esser convenia da sé lucente
	quel ch’era dentro al sol dov’ io entra’mi,
non per color,   ma per lume parvente!
