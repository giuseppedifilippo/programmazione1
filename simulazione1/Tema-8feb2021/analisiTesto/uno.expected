Bëatrice : 1
Quant’ : 1
al : 1
bene : 1
che : 2
ch’era : 1
color, : 1
convenia : 1
da : 1
dentro : 1
di : 1
dov’ : 1
entra’mi, : 1
esser : 1
in : 1
io : 1
lucente : 1
lume : 1
l’atto : 1
ma : 1
meglio, : 1
non : 2
parvente! : 1
per : 3
quel : 1
quella : 1
scorge : 1
si : 1
sol : 1
sporge. : 1
subitamente : 1
suo : 1
sé : 1
sì : 2
tempo : 1
È : 1
1 3 1.1388888888888888
[scorge si sol sporge. subitamente suo sé sì]
